### ISSUE TITLE
Cannot confirm after removing tokens from a task

### ISSUE THREAD

**Issue by auryn-macmillan** [JoinColony/colonyDapp#2037]
## Steps to reproduce

1. Create a task
2. Allocate some funds to it
3. Try to remove the allocated funds

## Expected behaviour

After clicking "remove", the "confirm" button should be active and apply the change when clicked.

## Actual behaviour

The "confirm" button is greyed out.
![remove funding](https://user-images.githubusercontent.com/8453294/74272002-df87f700-4cdb-11ea-9e22-44bb179958ee.gif)

**Comment by rdig** [comment]
Yup, this is known, we just didn't get to it yet. I'll leave this open, 'till we fix it.
